Userland/Libraries/LibWeb/PixelUnits.h

```diff
@@ -248,6 +248,86 @@ constexpr CSSPixels operator/(CSSPixels left, unsigned long right) { return left
 inline float operator/(CSSPixels left, float right) { return left.to_float() / right; }
 inline double operator/(CSSPixels left, double right) { return left.to_double() / right; }
 
+inline CSSPixels& operator*=(CSSPixels& left, float right)
+{
+    left = left.to_float() * right;
+    return left;
+}
+inline CSSPixels& operator*=(CSSPixels& left, double right)
+{
+    left = left.to_double() * right;
+    return left;
+}
```

Comment:
I would argue that it might be better if conversion and operations between `float` and `CSSPixels` should _always_ be explicit rather than adding more implicit ones, since that will encourage people to find other solutions than resorting to floating point. Converting to floating point may result in a loss of precision, as well as potentially being (probably very slightly) costly to performance vs remaining in fixed point.

Reply:
I already had to do some work in the past to get rid of implicit conversions and I agree with that. @MacDue could we instead use explicit functions for conversions?